elixir/apps/web/lib/web/live/users/edit.ex

```diff
@@ -1,65 +1,52 @@
 defmodule Web.Users.Edit do
```

Comment:
@bmanifold you need to rename modules and folder name to actors too

Reply:
Ah, good catch!

Reply:
This has been updated.